plugins/autoid/plugin.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 (function () {
+  var ALPHABET = '23456789abdegjkmnpqrvwxyz';
```

Comment:
see the reffed blogpost, but this basically eliminates the possibility of ambiguous characters and accidental bad words